api/v1/client/daemon/daemon_client.go

```diff
@@ -48,7 +48,7 @@ type ClientService interface {
 }
 
 /*
-  GetClusterNodes gets nodes information stored in the cilium agent
```

Comment:
Almost all `.go` files in `api/v1` are auto-generated. Are these formatting changes due to the builder base image change or did you explicitly gofmt those files? If the latter, then they'll likely get undone if you run `make -C api/v1` again.

Reply:
These got `gofmt`ed like that when running `contrib/scripts/check-api-code-gen.sh` (which in turn runs `make -C api/v1`) due to the builder image being update to Go 1.19, it seems. This is also what's run as part of CI in the `generate-api` step of the `lint-go` workflow.